x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/index.tsx

```diff
@@ -477,3 +477,29 @@ const IntegrationBreadcrumb: React.FunctionComponent<{
   useBreadcrumbs('add_integration_to_policy', { pkgTitle, pkgkey });
   return null;
 };
+
+const useNavigateToCallback = () => {
```

Comment:
If we end up needing this type of processing in the other page (Edit Integration Policy), can then move this to a more re-usable location